lib/Rest/GeneralResponse.cpp

```diff
@@ -419,6 +419,7 @@ rest::ResponseCode GeneralResponse::responseCode(int code) {
     case TRI_ERROR_USER_DUPLICATE:
     case TRI_ERROR_TASK_DUPLICATE_ID:
     case TRI_ERROR_GRAPH_DUPLICATE:
+    case TRI_ERROR_CLUSTER_FOLLOWER_TRANSACTION_COMMIT_PERFORMED:
```

Comment:
I would have classified this as SERVER_ERROR not CONFLICT

We do not conflict with any of the existing data in my opinion.
This server is now in a situation that it cannot solve by itself and cannot be solved by "trying again after update" we need to somehow modify the servers environment now.

Reply:
It should only be seen internally though